Makefile

```
@@ -44,7 +44,11 @@ lint:  ## Run a `lint` process on Hera and report problems
 
 .PHONY: test
 test:  ## Run tests for Hera
-	@poetry run python -m pytest --cov-report=term-missing -m "not on_cluster"
+	@poetry run python -m pytest --cov-report=term-missing -m "not on_cluster and not cli"
+
+.PHONY: test-cli
+test-cli:  ## Run tests for Hera
+	@poetry run python -m pytest --cov-report=term-missing -m 'cli'
```

Wouldn't recommend a coverage report for a subset of tests. Might be worth a new `test-all` make target for regular tests and cli
```suggestion
	@poetry run python -m pytest -m 'cli'
```